po/extra/fr.po

```diff
@@ -44,45 +44,97 @@ msgid ""
 msgstr ""
```

Comment:
I don't expect all source strings are always translated so I just comment to ask, but is it just you forgot to translate or you have some difficulties to translate this string (or another reason) that this line is not being translated?

Reply:
You're right I just forgot this one. I'll translate it. Thanks for the report :smile:!